testSuite/js/src/test/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -0,0 +1,27 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.utils
+
+import scala.scalajs.runtime
+
+object Platform {
+  /** Returns `true` if and only if the code is executing on a JVM.
+   *  Note: Returns `false` when executing on any JS VM.
+   */
+  final val executingInJVM = false
+
+  final val executingInJVMOnJDK6 = false
+
+  final val executingInJVMOnJDK7OrLower = false
+
+  // Members that are only accessible from testSuite/js
+  // (i.e. do no link on the JVM).
+
+  def areTypedArraysSupported: Boolean =
+    runtime.Bits.areTypedArraysSupported
```

Comment:
I guess `areTypedArraysSupported` will not be needed in the context of `java.util.logging`